materialized_views_test.py

```diff
@@ -2559,6 +2562,76 @@ def _test_base_view_consistency_on_crash(self, fail_phase):
             assert base_entry, "Both base {} and view entry {} should exist.".format(base_entry, view_entry)
             assert view_entry, "Both base {} and view entry {} should exist.".format(base_entry, view_entry)
 
+    def test_create_view_on_node_down(self):
+        """
+        Test view schema propagation when node is down and started
+        @jira_ticket: STAR-690
+        """
+        self.prepare(rf=1, nodes=2, options={'hinted_handoff_enabled': False})
+        node1, node2 = self.cluster.nodelist()
+        rows = 100
+
+        session1 = self.patient_exclusive_cql_connection(node1, keyspace="ks")
+
+        logger.debug("populate data on base table")
+        session1.execute("CREATE TABLE cf (pk int, ck int, a int, b int, primary key(pk,ck))")
+        for i in range(rows):
+            session1.execute("INSERT INTO cf(pk,ck,a,b) VALUES({},{},{},{})".format(i, i, i, i))
+
+        logger.debug("stop node2")
+        node2.stop(wait_other_notice=True)
+
+        logger.debug("create mv on node1")
+        session1.execute(("CREATE MATERIALIZED VIEW mv AS SELECT * FROM cf "
+                          "WHERE pk IS NOT NULL AND ck IS NOT NULL PRIMARY KEY (ck, pk)"))
+        session1.cluster.control_connection.wait_for_schema_agreement()
+
+        logger.debug("start node2, wait for view built")
+        node2.start(wait_other_notice=True)
+        session2 = self.patient_exclusive_cql_connection(node2, keyspace="ks")
+        self._wait_for_view('ks', 'mv', delay=30)
+
+        logger.debug("Verify view data")
+        assert_one(session2, "SELECT count(*) FROM mv", [rows])
+        for i in range(rows):
+            assert_one(session2, "SELECT * FROM cf WHERE pk={} AND ck={}".format(i, i), [i, i, i, i])
+            assert_one(session2, "SELECT * FROM mv WHERE pk={} AND ck={}".format(i, i), [i, i, i, i])
+
+    def test_alter_view_on_node_down(self):
```

Comment:
actually there is no ALTER VIEW here...
rather `view_is_automatically_altered_on_table_change_on_node_down` :-)

Reply:
no need to change if you think it does not make sense or if we want to keep oss compatible test names.